scrapers/nus-v2/src/tasks/CollateModules.ts

```diff
@@ -118,10 +118,12 @@ const getModuleCondensed = ({
   moduleCode,
   title,
   semesterData,
+  attributes,
 }: ModuleWithoutTree): ModuleCondensed => ({
   moduleCode,
   title,
   semesters: semesterData.map((semester) => semester.semester),
+  yearLong: Boolean(attributes ? attributes.year : false),
```

Comment:
`Boolean(undefined) === false` 

```suggestion
  yearLong: Boolean(attributes.year),
```